package.json

```diff
@@ -18,11 +18,12 @@
     "coverage": "nyc npm test && npm run generate-lcov && npm run coveralls-coverage && npm run codeclimate-coverage",
     "codeclimate-coverage": "codeclimate-test-reporter < lcov.info",
     "generate-lcov": "nyc report --reporter=text-lcov > lcov.info",
-    "test": "cross-env NODE_ENV=test mocha --require @babel/register src/test/**.test.js --timeout 20000 --exit"
+    "test": "cross-env NODE_ENV=test npm run heroku-postbuild && nyc _mocha --require @babel/register src/test/**.test.js --timeout 20000 --exit"
```

Comment:
Ok.. do we have to run Heroku post-build every time we run a test? not really sure tho

Reply:
The script is to doe fresh migration every time you want to test...so Yes, I have to